src/main/resources/javascript/locales/en.json

```diff
@@ -209,7 +209,7 @@
           "system": {
             "displayName": "System",
             "fields": {
-              "systemNameDescription": "{{maxNameSize}} characters maximum, special characters will be ignored. Used to build the URL for pages if no vanity URL is set in advanced options.",
```

Comment:
What about other i18n files ? I can see they still mention what you have deleted here for "en".